src/main/java/com/github/jparkie/pdd/BitArray.java

```diff
@@ -38,22 +38,35 @@ private static int numWords(long numBits) {
         return (int) numWords;
     }
 
+    private final class indexAndMask {
```

Comment:
Class name should be CamelCase. Since this inner class doesn't need to rely on the outer scope. Making it a `private static final class` would be better.